ads/google/a4a/utils.js

```diff
@@ -106,16 +105,17 @@ export function extractGoogleAdCreativeAndSignature(
  * @param {!../../../extensions/amp-a4a/0.1/amp-a4a.AmpA4A} a4a
  * @param {string} baseUrl
  * @param {number} startTime
- * @param {number} slotNumber
  * @param {!Array<!./url-builder.QueryParameterDef>} queryParams
  * @param {!Array<!./url-builder.QueryParameterDef>} unboundedQueryParams
  * @param {(string|undefined)} clientId
  * @param {string} referrer
  * @return {string}
  */
 function buildAdUrl(
```

Comment:
Is there any real value in googleAdUrl and buildAdUrl being separate functions?  I believe buildAdUrl can be private